templates/includes/card-group-record-summary.html

```diff
@@ -10,7 +10,7 @@
                     {{ record.reference_number }} -
                 </span>
                 {% endif %}
-                {{ record.title }}
+                {{ description_override|default:record.description|safe }}
```

Comment:
We shouldn't ever mark anything outside of our control as safe (e.g. user input / api content), so would be good to update this. Maybe `firstof` would work here?

```suggestion
                {% firstof description_override|safe record.description|striptags|escape %}
```